assets/json-schema-faker.js

```diff
@@ -1446,8 +1446,8 @@
   var clone = module.exports = function(obj, seen) {
     seen = seen || [];
 
-    if (seen.indexOf(obj) > -1) {
-      throw new Error('unable dereference circular structures');
+    if (seen.indexOf(obj) > -1 && seen[indexOf(obj)].repetition > 2) {
+      return 'unable dereference circular structures';
```

Comment:
Deref.js returns ` return { value: '<Error: Too many levels of nesting to fake this schema>' };`

We should use a similar sentence structure. Create a schema with >20 levels of nesting, and one with >2 levels of repetition - the error structure should be the same.

Also, what is `seen[indexOf(obj)]`? Need to add tests for the failing case as well.